### DISCUSSION TITLE
Quote Icon

### DISCUSSION THREAD

**Discussion by sasial-dev** [tailwindlabs/heroicons#506] [Ideas]
A block-style **"**

**Comment by ndri** [comment]
I added `quotation-mark` and `blockquote` to [Sidekickicons](https://github.com/ndri/sidekickicons/), which are meant to be used as a companion to Heroicons. Browse them here:
- https://sidekickicons.com/?icon=blockquote
- https://sidekickicons.com/?icon=quotation-mark
